backend/backend/settings/base.py

```diff
@@ -143,6 +143,14 @@ def get_required_setting(
 
 DEFAULT_AUTH_USERNAME = os.environ.get("DEFAULT_AUTH_USERNAME", "unstract")
 DEFAULT_AUTH_PASSWORD = os.environ.get("DEFAULT_AUTH_PASSWORD", "unstract")
```

Comment:
@muhammad-ali-e even these in my opinion we should use `get_required_setting` and remove the default values. cc: @hari-kuriakose